Description: Declare ISO-8859-1 encoding in texinfo file
 Since texinfo 6.7 the default encoding is UTF-8,
 which leads to an error.
Author: Tobias Hansen <thansen@debian.org>

--- a/interfaces/emacs/imaxima/imaxima.texi
+++ b/interfaces/emacs/imaxima/imaxima.texi
@@ -1,4 +1,5 @@
 \input texinfo                  @c -*-texinfo-*-
+@documentencoding ISO-8859-1
 @c %**start of header
 @setfilename imaxima.info
 @settitle Imaxima Manual
